preppal-be/models/calendar.ts

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const date = new Date().toString();
+
+const CalendarSchema = new Schema({
+    userName: {
```

Comment:
change this to username

variables don't match here
```            
const newCalendar = await new Calendar({
                username,
                calendarDates
            })
```
which could cause some problems